frontend/src/views/WheresMyGene/components/HeatMap/utils.ts

```diff
@@ -30,7 +27,7 @@ const COMMON_OPTIONS = {
 };
 
 const COMMON_SERIES: ScatterSeriesOption = {
-  emphasis: { itemStyle: { color: "inherit" }, scale: false },
+  emphasis: { itemStyle: { color: "inherit" }, scale: true },
```

Comment:
just curious, is the default for `scale` `true`? If so, we can probably just remove `scale: true` altogether?